src/test/kotlin/com/terraformation/backend/accelerator/ParticipantProjectSpeciesServiceTest.kt

```diff
@@ -57,14 +62,15 @@ class ParticipantProjectSpeciesServiceTest : DatabaseTest(), RunsAsUser {
           insertDeliverable(moduleId = moduleId, deliverableTypeId = DeliverableType.Species)
       insertCohortModule(cohortId = cohortId, moduleId = moduleId)
 
-      service.create(
-          NewParticipantProjectSpeciesModel(
-              feedback = "feedback",
-              id = null,
-              modifiedTime = Instant.EPOCH,
-              projectId = projectId,
-              rationale = "rationale",
-              speciesId = speciesId))
+      val existingModel =
```

Comment:
Would it make sense to assert that this model looks like it's supposed to, either in addition to or instead of the assertion on `SubmissionsRow`?

Reply:
I figured that was covered adequately by the store tests, but certainly wouldn't hurt to add it here